### PR TITLE
Do not send reset password email to disabled user

### LINKED ISSUE #59: Password reset request to inactive user
obviously this should not happen :)

### PR DESCRIPTION
**Pull request by szymach**
Resolves #59, after #87

### CODE REVIEW

File: Controller/PasswordReset/ResetRequestController.php [hunk 1]
```diff
@@ -93,15 +93,18 @@ public function __construct(
...
-        if ($form->isValid()) {
+        if ($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
 
             $user = $this->getUser($form);
             if (!($user instanceof ResettablePasswordInterface)) {
                 return $this->addFlashAndRedirect('info', 'admin.password_reset.request.mail_sent_if_correct');
             }
 
+            if (!$user->isEnabled()) {
+                return $this->addFlashAndRedirect('warning', 'admin.password_reset.request.user_disabled');
```
Comment (chives) [inline Controller/PasswordReset/ResetRequestController.php]: this is a security issue as long as it allows to detect user existence. this information should be sent via e-mail :)

Comment (szymach) [inline Controller/PasswordReset/ResetRequestController.php]: Indeed, though currently it is done similarly done in other cases. I would leave it as it is for now and do another PR that changes all cases.

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@rn0 @JarekW
